encfs 1.9 on macOS: Undefined symbol   "_libintl_gettext"

Full build log: https://gist.github.com/ilovezfs/0fdc404e5fcb55c9c3e006f17fbcc38c

```
[ 36%] Linking CXX executable checkops
/usr/local/Cellar/cmake/3.6.2/bin/cmake -E cmake_link_script CMakeFiles/checkops.dir/link.txt --verbose=1
/usr/local/Library/Homebrew/shims/super/clang++   -DNDEBUG -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk -mmacosx-version-min=10.11 -Wl,-search_paths_first -Wl,-headerpad_max_install_names  CMakeFiles/checkops.dir/encfs/test.cpp.o  -o checkops  -L/tmp/encfs-20160911-20518-1pii625/encfs-1.9/build/internal/tinyxml2-3.0.0  libencfs.a /usr/local/lib/libosxfuse.dylib /usr/local/opt/openssl/lib/libssl.dylib /usr/local/opt/openssl/lib/libcrypto.dylib internal/tinyxml2-3.0.0/libtinyxml2.a -Wl,-rpath,/tmp/encfs-20160911-20518-1pii625/encfs-1.9/build/internal/tinyxml2-3.0.0 
Undefined symbols for architecture x86_64:
  "_libintl_gettext", referenced from:
      encfs::userAllowMkdir(int, char const*, unsigned short) in libencfs.a(FileUtils.cpp.o)
      encfs::createV6Config(encfs::EncFS_Context*, std::__1::shared_ptr<encfs::EncFS_Opts> const&) in libencfs.a(FileUtils.cpp.o)
      encfs::showFSInfo(encfs::EncFSConfig const*) in libencfs.a(FileUtils.cpp.o)
      encfs::EncFSConfig::getUserKey(bool) in libencfs.a(FileUtils.cpp.o)
      encfs::EncFSConfig::getUserKey(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) in libencfs.a(FileUtils.cpp.o)
      encfs::EncFSConfig::getNewUserKey() in libencfs.a(FileUtils.cpp.o)
      encfs::EncFSConfig::makeKey(char const*, int) in libencfs.a(FileUtils.cpp.o)
      ...
ld: symbol(s) not found for architecture x86_64
```
As a workaround for Homebrew in https://github.com/Homebrew/homebrew-fuse/pull/65 I'm doing the following:
```
ENV.append "LDFLAGS", "-lintl"
```

But it would be great if this regression could be fixed upstream so that the above hack isn't needed.

Thanks!

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -123,6 +123,11 @@ include_directories (${CMAKE_SOURCE_DIR})
 
 # Translations
 if (ENABLE_NLS)
+  find_package (Intl)
+  if (Intl_FOUND)
+    include_directories (${Intl_INCLUDE_DIRS})
+  endif()
+
   add_subdirectory(po)
 
   add_definitions(-DENABLE_NLS)
@@ -168,6 +173,7 @@ target_link_libraries(encfs
   ${OPENSSL_LIBRARIES}
   ${TINYXML_LIBRARIES}
   ${CMAKE_THREAD_LIBS_INIT}
+  ${Intl_LIBRARIES}
 )
 if (INSTALL_LIBENCFS)
   install (TARGETS encfs DESTINATION lib)

